tests/chimera-open/snitchCluster/simpleOffload/src_cluster/test_cluster.c

```diff
@@ -9,35 +9,33 @@
 
 #include "soc.h"
 
-static uint32_t *clintPointer = (uint32_t *)CLINT_CTRL_BASE;
+#include "trampoline_snitchCluster.h"
+
+uint32_t *clintPointer = (uint32_t *)CLINT_CTRL_BASE;
 
 /**
  * @brief Interrupt handler for the cluster, which clears the interrupt flag for the current hart.
  *
  * @warning Stack, thread and global pointer might not yet be set up!
  */
 __attribute__((naked)) void clusterInterruptHandler() {
-    asm volatile(
-        // Load global pointer
-        ".option push\n"
-        ".option norelax\n"          // Disable relaxation to ensure `la` behaves as expected
-        "la gp, __global_pointer$\n" // Load address of global pointer
-        ".option pop\n"
-
-        // Set thread pointer (tp) to zero
-        "mv tp, zero\n"
+    SNITCH_SETUP_CORE();
 
+    asm volatile(
         // Load mhartid CSR into t0
         "csrr t0, mhartid\n"
         // Load the base address of clintPointer into t1
-        "lw t1, %0\n"
+        // "lw t1, %0\n"
+        "lui t1, %%hi(clintPointer)\n"
+        "addi t1, t1, %%lo(clintPointer)\n"
```

Comment:
Does `la clintPointer`, or better yet `la __base_clint` not work? Using linker symbols for address offsets reduces sources of truth, imo, and let's you use simplified assembly.